package.json

```diff
@@ -97,6 +98,7 @@
     "pg": "*",
     "prettier": "^2.2.1",
     "prom-client": "*",
+    "proxyquire": "^2.1.3",
```

Comment:
🙃 Note that we also use mock-require in this repo for the same thing.

Reply:
I know, but have you seen this (https://github.com/boblauer/mock-require)
![Screenshot 2021-11-18 at 12 23 42 PM](https://user-images.githubusercontent.com/44189854/142397787-210aff2a-d0d1-46e5-b396-52da754f666c.png)
?

Reply:
😮 we should remove mock-require then at some point